core-services/image-mirroring/knative/mapping_knative_knative-v1.7_eventing-kafka-broker_quay

```diff
@@ -0,0 +1,22 @@
+registry.ci.openshift.org/openshift/knative-eventing-kafka-broker-dispatcher quay.io/openshift-knative/knative-eventing-kafka-broker-dispatcher
```

Comment:
I can't pull these

Reply:
This works for me: docker pull registry.ci.openshift.org/openshift/knative-eventing-kafka-broker-dispatcher:knative-v1.7

Reply:
this syntax is omitting the tag to mirror every tag

Reply:
OK, so the mirror job knows this, and does than apply the version?

Reply:
I have some love for explicit versions, since a string like `registry.ci.openshift.org/openshift/knative-eventing-kafka-broker-dispatcher:knative-v1.7` is a bit easier for debugging, since I can just use it, instead of copy the string, and manually add something to it for testing.

But if the mirror job does this, than I am OK to do all implicit versions, and we fiddle the rest manually, if we need to debug something. Not a fan, but OK

Reply:
This is mirroring commit tags, we cannot reference every commit here

Reply:
Ah, check - ok. sorry, now I get it